lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
+## <Package>Sonata</Package>:
+##  <P/>
+##  <Log><![CDATA[
+##  gap> LoadPackage( "Sonata" );
+##  #I  You may wish to install the xgap package
+##  #I  and enjoy the graphic capabilities of SONATA.
+##
+##   ___________________________________________________________________________
+##  /        ___
+## ||       /   \                 /\    Version 2.6
+## ||      ||   ||  |\    |      /  \               /\       Erhard Aichinger
+##  \___   ||   ||  |\\   |     /____\_____________/__\      Franz Binder
+##      \  ||   ||  | \\  |    /      \     ||    /    \     Juergen Ecker
+##      ||  \___/   |  \\ |   /        \    ||   /      \    Peter Mayr
+##      ||          |   \\|  /          \   ||               Christof Noebauer
+##  \___/           |    \|                 ||
+##
+##  System    Of   Nearrings     And      Their Applications
+##  Info: http://www.algebra.uni-linz.ac.at/Sonata/
+##
+##  true
+##
```

Comment:
Extra empty line may be removed. What about adding an example of using abbreviated name of the package? For example, `LoadPackage("semi");` will load Semigroups package, or even `LoadPackage("d");` will load DESIGN.